templates/go/cloudevents/README.md

```diff
@@ -9,13 +9,14 @@ Develop new features by adding a test to [`handle_test.go`](handle_test.go) for
 Update the running analog of the function using the `func` CLI or client library, and it can be invoked using a manually-created CloudEvent:
 
 ```console
-curl -X POST -d '{"hello": "world"}' \
+curl -v -X POST -d '{"input": "hello"}' \
   -H'Content-type: application/json' \
   -H'Ce-id: 1' \
   -H'Ce-source: cloud-event-example' \
-  -H'Ce-type: dev.knative.example' \
+  -H'Ce-subject: Convert to UpperCase' \
+  -H'Ce-type: UppercaseRequestedEvent' \
   -H'Ce-specversion: 1.0' \
-  http://myFunction.example.com/
+  http://localhost:8080/
```

Comment:
@lkingland regarding the repo: https://github.com/boson-project/templates can we move those out from that org? I bet that this has been brought up before.. how much effort would that take? Do we have an issue for that?

Reply:
+10 for invoke!

Reply:
+1

Reply:
> @lkingland regarding the repo: https://github.com/boson-project/templates can we move those out from that org? I bet that this has been brought up before.. how much effort would that take? Do we have an issue for that?

@salaboy we have talked about moving that, but there has been mostly inertia on the subject. When the original move from `boson-project/func` to `knative-sandbox/kn-plugin-func` happened, the idea was to keep the move minimal and targeted. Now, however, with a solid cadence and regular contributions stemming from different companies, it may be time to revisit this. We can discuss at the next task force call, and go from there, wdyt?

Reply:
> +10 for invoke!

I agree, and yet not. :smile_cat: A nice thing about `curl` that is different than `invoke` is an ability to see the response and headers.

Reply:
I think the ability to see the response and headers is quite critical to this PR.. as the function will be returning a CloudEvent that I want to see when I execute the command

Reply:
> regarding the repo: https://github.com/boson-project/templates can we move those out from that org? I bet that this has been brought up before.. 

It has not been brought up before.  Therefore I brought it up today in the meeting because it is a good suggestion.  Lance graciously offered to create the issue which initiates the process.

It will be nice to have a set of official examples; especially to compliment docs.

Reply:
> > +10 for invoke!
> 
> I agree, and yet not. 😸 A nice thing about `curl` that is different than `invoke` is an ability to see the response and headers.

You know.... `invoke --verbose` should do this.